source/features/widen-search-field.css

```diff
@@ -13,26 +13,6 @@ https://github.com/sindresorhus/refined-github/milestones
 https://github.com/sindresorhus/refined-github/releases/tag/19.9.16
 */
 
-/* Expand issue search field width */
-.repository-content [role='search'], /* Repo issue/PR lists */
-.subnav-links + .float-right, /* Global search pages */
-.subnav-search,
-.subnav-search-input {
-	flex-grow: 1;
-	display: flex;
-	align-items: flex-start;
-}
-
-/* Clear .subnav-search input field when using display: flex */
-form.subnav-search > div.card-filter-autocomplete-dropdown {
-	margin-top: 40px !important;
-}
```

Comment:
The above two declarations are no longer needed since they are already full width of the container.